VortexEngine/src/patterns/multi/DripMorphPattern.cpp

```diff
@@ -1,5 +1,7 @@
 #include "DripMorphPattern.h"
 
+// NOTE: I'm not sure this is right but I'm replicating the recent changes to Blinkstep here
```

Comment:
Can dripmorph be written as a BlinkStep pattern? if not then this is fine

Reply:
I feel like it can be, should I convert it to a blinkstep pattern?

Reply:
I don't think it can, I think that's why we didn't do it.

Reply:
feel free to try, doesn't matter

Reply:
meh XD if it works it works i guess